### DISCUSSION TITLE
Document Access Problem with Web Part Search Results

### DISCUSSION THREAD

**Discussion by DamienLortieThibaut** [microsoft-search/pnp-modern-search#3842] [Q&A]
Hi everyone,

I have an access problem with data from a particular SharePoint site. Indeed, in the "Query Template" section when I specify with Path the data I wish to display. I can display data from external sites belonging to the same tenant but documents from sites to which PnP Search is attached, I always have an empty result.

Does anyone know where the problem could be coming from?

Thanks to those who can answer me,
Have a nice day !

**Comment by kasperbolarsen** [comment]
You guess you can access the site and the docs, which makes me think of two options:
1) The site and/or the library are set to NOT be indexed by the search engine or
2) the content hasn't been indexed yet (sometime is takes like forever)

Just to verify your query, could you use the query in the Microsoft search box on the tenant level?
